packages/cli/src/commands/setup/auth/providers/firebase.js

```diff
@@ -1,13 +1,12 @@
 // the lines that need to be added to App.{js,tsx}
 export const config = {
   imports: [
```

Comment:
I think the linter wants you to put your imports on one line, now that they're shorter
```
imports: [
  "import firebase from 'firebase/app", "import 'firebase/auth'"
]
```

Reply:
Pushed a fix, let's see how it goes.
Actually I had setup a pre-commit with yarn --eslint, it worked for a while then it began taking a long time until eventually breaking up. I'll set it up again later :) I've got some clean up to do of my env!